### ISSUE TITLE
50.000 queries to wa2-mz36-qrmzh6.bosch.de

### ISSUE THREAD

**Issue by bschokker** [robertklep/nefit-easy-http-server#45]
Dear Robert,
Last weekend I've updated my Domoticz to the latest version on a Raspberry Pi.
The same I did for the "nefit-easy-http-server" and all is working fine.
Today I saw on my pi-hole extremely many requests to the domain wa2-mz36-qrmzh6.bosch.de. These requests are stopping immediately as soon I stop the "nefit-easy-http-server"
Is there something that I can do to decrease the amount of queries to bosch.de?

Many thanks in advance.

Kind regards

B.S.

**Comment by robertklep** [comment]
> Is there something that I can do to decrease the amount of queries to bosch.de?

The Easy server makes a single persistent connection to the Bosch backend (the hostname you see appear in the logs) when it starts, so it doesn't really make a lot of sense that you see that many requests being made (especially since you also say it's working fine, because the only reason I can think of for that many requests to be made is if the server keeps getting restarted).

Can you see the type of requests?

**Comment by bschokker** [comment]
The type of requests that I see in the pi-hole are A and AAAA.
Almost every 5 second I see them in a pair A - AAAA
One thing is sure, this was in the past not the case because I look on a frequent base to the pi-hole.

**Comment by robertklep** [comment]
Can you check and see if the Easy server isn't restarting?

Perhaps Domoticz itself is performing those lookups, I don't use it myself so no idea if that might be the case.

**Comment by bschokker** [comment]
The Easyserver isn't restarting, is up and running without any problems.
I will check the network configuration in the Pi, I updated from Jessie to Buster, maybe there is something changed. I let you know.

**Comment by robertklep** [comment]
I wonder if it's actually working or if you're running into this problem: https://github.com/robertklep/nefit-easy-http-server#problems-on-recent-linux-distributions (and the server is _running_, but it keeps trying to connect).

**Comment by bschokker** [comment]
It's working that is for sure. I can change the temperature set point in Domoticz and my thermostat is responding in a split second . I take this evening a look to the topic you mentioned about recent Linux distributions.

**Comment by bschokker** [comment]
I'm afraid he's trying to keep connecting. After a netstat I see this:

tcp        0      0 domoticz-master.f:48586 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 domoticz-master.f:48540 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 domoticz-master.f:48570 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 domoticz-master.f:41070 wa2-mz36-qr:xmpp-client ESTABLISHED
tcp        0      0 domoticz-master.f:48584 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 domoticz-master.f:48530 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 domoticz-master.f:48556 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 domoticz-master.f:48560 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 domoticz-master.f:48554 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 domoticz-master.f:48558 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 localhost:43424         localhost:3000          TIME_WAIT  
tcp        0      0 domoticz-master.f:48532 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 localhost:43386         localhost:3000          TIME_WAIT  
tcp        0      0 domoticz-master.f:48590 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 domoticz-master.f:36096 domoticz-slave.fri:6144 ESTABLISHED
tcp        0      0 localhost:43426         localhost:3000          TIME_WAIT  
tcp        0      0 domoticz-master.f:48562 wa2-mz36-qr:xmpp-client TIME_WAIT  
tcp        0      0 localhost:43420         localhost:3000          TIME_WAIT  
tcp        0      0 domoticz-master.f:48594 wa2-mz36-qr:xmpp-client ESTABLISHED

So you are right, it's some kind of loop he's in.

I already changed the ssl config file to:

MinProtocol = None
CipherString = DEFAULT

**Comment by robertklep** [comment]
How do you start/run `easy-server`? Try running it manually from an ssh session:
```
$ env DEBUG=* easy-server --serial=XXX --access-key=XXX --password=XXX
```

That should log a whole bunch of stuff, perhaps there's a clue why it's reconnecting.

**Comment by bschokker** [comment]
It is solved, I restored the formal working domoticz (jessie) to a new SD card, incl. the working nefit-easy-server to check some configuration files. After that the SD card with buster again and the nefit-easy-server starts automatically. After a netstat I saw that he connection was established without reconnecting. No idea why but it seems he needed a complete shutdown ?

Here the netstat output:
tcp        0      0 domoticz-master.f:40390 wa2-mz36-qr:xmpp-client ESTABLISHED

Also the pi-hole doesn't show any queries anymore.

Thanks for all !

**Comment by bschokker** [comment]
t is solved, I restored the formal working domoticz (jessie) to a new SD card, incl. the working nefit-easy-server to check some configuration files. After that the SD card with buster again and the nefit-easy-server starts automatically. After a netstat I saw that he connection was established without reconnecting. No idea why but it seems he needed a complete shutdown ?

Here the netstat output:
tcp 0 0 domoticz-master.f:40390 wa2-mz36-qr:xmpp-client ESTABLISHED

Also the pi-hole doesn't show any queries anymore.

Thanks for all !
